src/pages/create-page.tsx

```diff
@@ -1,7 +1,30 @@
 const CreatePage = () => {
   return (
-    <main>create-page</main>
-  )
-}
+    <main>
+      <h1>Crie uma nota</h1>
 
-export {CreatePage}
\ No newline at end of file
+      <form action="post" className="form">
+        <label htmlFor="title">
+          Título
+          <input type="text" name="title" id="title" className="form__title" />
+        </label>
```

Comment:
```
<label htmlFor="title">
  Título
  <input type="text" name="title" id="title" className="form__title" />
</label>
```
O `for` da _tag_ `label` só será usado quando _label_ e _input_ forem irmãos. Você escolheu envolver o _input_ com a tag _label_ e isso é válido desde que sejam feitas as devidas alterações.